core-java-modules/core-java-21/src/test/java/com/baeldung/java21/PatternCaseLabelsUnitTest.java

```diff
@@ -0,0 +1,48 @@
+package com.baeldung.java21;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.junit.jupiter.api.Test;
+
+public class PatternCaseLabelsUnitTest {
+
+    @Test
+    public void whenProcessInputOldWayWithYes_thenReturnOutput() {
```

Comment:
Using JUnit 5 we don't need to make class nor method public.

Reply:
Done